lib/router/transition.js

```diff
@@ -95,6 +101,8 @@ Transition.prototype = {
   },
 
   /**
+    @property promise
```

Comment:
These declarations have to come after the description (otherwise YUIDoc will not parse the description at all).

So it should be:

```javascript
/**
  Description here

  <yuidoc statements here>
*/
```